src/main/java/com/google/sps/servlets/UserDataServlet.java

```diff
@@ -27,44 +27,33 @@
 @WebServlet("/user")
 public class UserDataServlet extends HttpServlet {
 
-  /** Responsible for creating new User. */
+  private DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+
+  /** Responsible for add a new User into Datastore */
```

Comment:
typo: *adding :)

Reply:
Yay! I totally agree! :)

Reply:
could even be changed to "Adds a new User into Datastore" haha

Reply:
Agree with @ubahl 

Prefer using active voice